packages/keystore_ffi/lib/generated.dart

```diff
@@ -1,12 +1,37 @@
 // AUTOMATICALLY GENERATED. DO NOT EDIT.
 
 import 'dart:ffi' as ffi;
+import 'dart:io' show Platform;
 import 'package:ffi/ffi.dart' as ffi;
 
+ffi.DynamicLibrary load({String basePath = ''}) {
+  if (Platform.isAndroid || Platform.isLinux) {
+    return ffi.DynamicLibrary.open('${basePath}libkeystore_ffi.so');
+  } else if (Platform.isIOS) {
+    return ffi.DynamicLibrary.process();
+  } else if (Platform.isMacOS) {
+    return ffi.DynamicLibrary.open('${basePath}libkeystore_ffi.dylib');
+  } else if (Platform.isWindows) {
+    return ffi.DynamicLibrary.open('${basePath}libkeystore_ffi.dll');
+  } else {
+    throw NotSupportedPlatform('${Platform.operatingSystem} is not supported!');
+  }
+}
+
+class NotSupportedPlatform implements Exception {
+  NotSupportedPlatform(String s);
+}
+
+ffi.DynamicLibrary dynamicLibrary() {
+  if (Platform.isWindows || Platform.isLinux || Platform.isMacOS) {
+    return load(basePath: '../../../target/debug/');
```

Comment:
mmh, can we avoid hardcoding paths?

Reply:
maybe Makefile.toml can set the env variables?

Reply:
yeah, I'm trying to find a way to not hard coding.
but the `ffi_helper` doesn't provide a way to a lib path for platform specifically.
so I had to hard code that, any idea to get workaround that in a better way?

Reply:
on linux you'd set `LD_LIBRARY_PATH` in `Makefile.toml`

Reply:
might have hit this problem which prevents setting env variables https://github.com/rust-lang/cargo/issues/4121

Reply:
> on linux you'd set LD_LIBRARY_PATH in Makefile.toml

https://doc.rust-lang.org/cargo/reference/environment-variables.html#dynamic-library-paths

Reply:
please note that this env is going to be on the target platform, not the host.
so I guess it's not gonna work this way!